prov/efa/test/efa_unit_test_use_device_rdma.c

```diff
@@ -0,0 +1,118 @@
+#include "efa_unit_tests.h"
+
+
```

Comment:
Also wonder about the code style. The max line length looks short - is that on purpose? Looks out of place.

Reply:
Line length is 80.  Was there something else about the style?